actions/image_partition_action.go

```diff
@@ -651,6 +651,8 @@ func (i *ImagePartitionAction) Verify(context *debos.DebosContext) error {
 		switch p.FS {
 		case "fat32":
 			p.FS = "vfat"
+		case "none", "None", "NONE":
```

Comment:
none is a valid filesystem type, this is wrong.

what you should probably meant to do is check for mounting something with a none mountpoint. So you should move this to the next `i.Mountpoints` for loop and change this to be something like:

```
if m.FS.ToLower() == "none" {
    return fmt.Errorf("Cannot mount filesystem with type '%s'", p.FS)
}
```

Reply:
@obbardc , Have updated the commit. Thanks.